### ISSUE TITLE
Can I know which one is the most likely orthologue in the multi-copy case?

### ISSUE THREAD

**Issue by jihoonjo** [davidemms/OrthoFinder#655]
Dear david.

In the Orthofinder results, the "SpeciesTreeAlignment.fa" contained a large protein alignment which is composed with each gene alignment by "STAG" algorithm.

Here, each gene list was not always single-copy orthogroup, some genes were multi-copy orthogroup due to the rareness of single-copy orthogroup in a large-scale ortholog analysis. And, the most likely orthologue among multi-copy genes was selected by the distance in their tree (these were mentioned in STAG algorithm paper). 

In the multi-copy case, however, I was not able to find any evidence about what orthologue was concatenated from each multi-copy orthogroup to the "SpeciesTreeAlignment.fa" in orthofinder results. 
Thus, what copy of them was concatenated? Can I track this selection process?

In other words, How to split the "SpeciesTreeAlignment.fa" file to each alignment which contains one protein sequence per one species?

Thank you in advance

**Comment by davidemms** [comment]
Hi

Details are here:
https://github.com/davidemms/OrthoFinder#multiple-sequence-alignment-species-tree-method--m-msa

If you have a "SpeciesTreeAlignment.fa" then it sounds like you used the "-M msa" option, in which case STAG isn't used. For the MSA option when an orthogroup is used with multiple genes from a particular species then then none of those genes are used. Genes are only used in a particular orthogroup if that species is single copy.

Best wishes
David

**Comment by jihoonjo** [comment]
Thanks, David!

I fully understood the process to make "SpeciesTreeAlignment.fa" from the details in your link.

But, I am still confused about 'how to split "SpeciesTreeAlignment.fa" to each gene alignment?'.
Because I need each gene alignment **that has one copy per one species.**

If some gene alignment in the fasta file includes like this example)

OG000001 
SpeciesA SpeciesB SpeciesC SpeciesD SpeciesE
     1copy       1copy     3copies      1copy      0copy

How to select the most orthologous copy among the 3 copies in species C?
or Are they should be removed?


Thank you in advance

**Comment by jihoonjo** [comment]
Thanks, David!

I fully understood the process to make "SpeciesTreeAlignment.fa" from the details in your link.

But, I am still confused about 'how to split "SpeciesTreeAlignment.fa" to each gene alignment?'.
Because I need each gene alignment that has one copy per one species.

If some gene alignment in the fasta file looks like this example (actually, some genes in my data also look like this).

OG000001
SpeciesA SpeciesB SpeciesC SpeciesD SpeciesE
1copy 1copy 3copies 1copy 0copy

How to select the most orthologous copy among the 3 copies in species C?
or Are they should be removed?

Thank you in advance

**Comment by davidemms** [comment]
Hi

For the MSA option when an orthogroup is used with multiple genes from a particular species then then none of those genes are used, that species is replaced by gaps, only the single-copy genes are used. 

For your example, the single copy genes from A, B & D are used and just blanks are used for C & E.

Best wishes
David

**Comment by jihoonjo** [comment]
Thank you, david! I'll take care of it.

I’m really grateful for your help :)
